### DISCUSSION TITLE
Spotube client wont start

### DISCUSSION THREAD

**Discussion by JeelChatrola** [KRTirtho/spotube#1572] [Q&A]
I had an older version of Spotube. When I opened the app, it prompted me with an update that was available for download.
I tried to install the updated version using sudo apt install <deb>. Now, the app won't start at all.

**Comment by FOSSisLife** [comment]
Hi! I noticed the same problem in my fedora 40. It's not exactly not starting, the app launches but just in background and not show the interface in foreground related to show/hide function in the app that remains flagged and we cannot change this. For fix temporaly I downgraded for version v3.6.0.
